tests/performance/test_performance.py

```diff
@@ -0,0 +1,178 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Can be run locally with: py.test tests/test_performance.py -v -s
+
+import os
+import subprocess
+
+import pytest
+import requests
+import datetime
+
+from tabulate import tabulate
+
+ROOT_PATH = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+DATASET = os.path.join(ROOT_PATH, "tests", "functional", "fixtures", "test_images", "performance")
+OVMS_CLIENT = os.path.join(ROOT_PATH, "example_client", "face_detection_performance.py")
+OVMS_DATASET = os.path.join(DATASET, "test_dir")
+
+CONFIG_4 = {"config": 4}
+CONFIG_32 = {"config": 32}
```

Comment:
It's not clear what this config means. Maybe we could change those two constants to a list:
`CORES_NUMBER = [4, 32]` ?